bin/service.js

```diff
@@ -90,6 +90,14 @@ rimraf.sync(argv.workDirectory);
 fs.mkdirSync(argv.taskDirectory);
 fs.mkdirSync(argv.workDirectory);
 
+// Copies the runtime node_modules to our task directory. This is necessary to make sure
+// we get the same depenencies available to us as in the Lambda environment.
+fs.cpSync(
```

Comment:
`service.js` is the entry point for the Docker container - so it runs every time the task executes. You'll see something similar in the `async_operations` update except that in the case of `async_operations` I _could_ do it in the Dockerfile vs at runtime.

The reason I had to do it here (every time the task runs) is because the "task directory" is configurable as a CLI option to the container. So we don't know where the Lambda will be unzipped to until runtime - so I copy the Lambda `node_modules` there at runtime.